tests/cli/multiphase_test.py

```diff
@@ -28,3 +28,22 @@ def test_view_matrix_command():
     )
     assert os.path.isfile(os.path.join(output_folder, 'skylight..class_room.vmx'))
     nukedir(output_folder)
+
+
+def test_daylight_matrix_command():
+    runner = CliRunner()
+    sender_file = './tests/assets/multi_phase/east_window_classroom..sender.rad'
+    receiver_file = './tests/assets/multi_phase/rflux_sky.sky'
+    input_oct = './tests/assets/multi_phase/scene.oct'
+    output_folder = './tests/assets/temp'
+    output = './tests/assets/temp/east_window_classroom.dmx'
+    nukedir(output_folder)
+    cmd_args = [
+        sender_file, receiver_file, input_oct,
+        '--rad-params', '-ab 3 -ad 1000 -lw 1e-05 -c 1000',
```

Comment:
-ab 3 might be too much for a test. I don't know how long it takes to run this command but it shouldn't take more than a few seconds for a test if you ask me.